modules/visual_script/visual_script_editor.cpp

```diff
@@ -2836,6 +2836,23 @@ void VisualScriptEditor::_graph_connected(const String &p_from, int p_from_slot,
 
 	ERR_FAIL_COND(from_seq != to_seq);
 
+	if (from_seq) {
+		// checking to prevent warnings
+		if (script->has_sequence_connection(p_from.to_int(), from_port, p_to.to_int())) {
+			return;
+		}
+	} else {
```

Comment:
Can you remove one layer of nested if statements?

Reply:
Sure - Done